openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -178,4 +179,4 @@ def patch(self, request, username):
                 status=status.HTTP_400_BAD_REQUEST
             )
 
-        return Response(status=status.HTTP_204_NO_CONTENT)
+        return Response(account_settings)
```

Comment:
Ok.  Thanks @catong.  The existing doc is correct for GET, but incorrect for PATCH (as currently implemented in production).  But *will be* correct once this code is in production.  :)